hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -86,11 +95,15 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
     });
     axios.all([
       axios.get('/api/v1/clusterState'),
-      axios.get('/api/v1/containers/missing')
-    ]).then(axios.spread((clusterStateResponse, missingContainersResponse) => {
+      axios.get('/api/v1/containers/missing'),
+      axios.get('/api/v1/task/status')
+    ]).then(axios.spread((clusterStateResponse, missingContainersResponse,taskstatusResponse) => {
+      
       const clusterState: IClusterStateResponse = clusterStateResponse.data;
-      const missingContainers: IMissingContainersResponse = missingContainersResponse.data;
+      const missingContainers: IMissingContainersResponse = missingContainersResponse.data;     
+      const taskStatus = taskstatusResponse.data && taskstatusResponse.data.filter((item:any) => item.taskName === 'OmDeltaRequest' || item.taskName === 'OmSnapshotRequest').sort((c1:any, c2:any) => c2.lastUpdatedTimestamp - c1.lastUpdatedTimestamp);
```

Comment:
Can we use two variables to hold values from `OmDeltaRequest` and `OmSnapshotRequest` separately so the logic can be cleaner? Thx.

Reply:
Will do respective changes